<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>demoForm</class>
 <widget class="QWidget" name="demoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Состояние ДЛС</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="refreshList">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="reversCheckBox">
          <property name="text">
           <string>Вывод с конца списка</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>Добавить</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Добавление узлов в ДЛС</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QRadioButton" name="toHead">
                  <property name="text">
                   <string>В начало списка</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="toTail">
                  <property name="text">
                   <string>В конец списка</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="beforePosition">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Перед выбранным</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="afterPosition">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>После выбраного</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="addPosition">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Добавить</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="addDemo">
                  <property name="text">
                   <string>Вставить демо-данные из базы</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Редактировать</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Редактирование узла ДЛС</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QPushButton" name="edit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Записать</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="remove">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Удалить</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QRadioButton" name="upRadioButton">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Переместить выше</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="downRadioButton">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Переместить ниже</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="toHeadRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Переместить в начало</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="toTailRadioButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Переместить в конец</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QPushButton" name="move">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Переместить</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="removeAll">
                <property name="text">
                 <string>Очистить весь список</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
